cdtb/tftdata.py

```diff
@@ -165,7 +169,23 @@ def parse_sets(self, map22, character_names, traits):
                 set_trait_paths += trait_lists[trait_list].getv("mTraits")
             set_traits = [traits[path] for path in set(set_trait_paths) if path in traits]
 
-            sets.append((set_number, set_mutator, set_name, set_characters, set_traits))
+            set_augment_ids = set()
+            set_items_ids = set()
+            for item_list in item.getv("ItemLists", []):
+                if item_list not in item_lists:
+                    continue
+                item_list_entry = item_lists[item_list]
+                for item_entry in item_list_entry.getv("mItems", []):
+                    if item_entry not in item_entries:
+                        continue
+                    if item_entries[item_entry].getv("IsAugment"):
+                        set_augment_ids.add(item_entries[item_entry].getv("mName"))
+                    else:
+                        set_items_ids.add(item_entries[item_entry].getv("mName"))
+            set_augments = list(set_augment_ids)
+            set_items = list(set_items_ids)
```

Comment:
```suggestion
            set_augments = sorted(set_augment_ids)
            set_items = sorted(set_items_ids)
```
JSON file will be a bit nicer for humans.

Reply:
Agreed. Changed.